src/lib/services/config-service.ts

```
@@ -250,12 +260,12 @@ export class ConfigService {
    * @returns Merged object
    */
   private static deepMerge<T>(target: T, source: Partial<T>): T {
-    const result = { ...target } as any;
+    const result = { ...target } as Record<string, unknown>;
     
     for (const key in source) {
       if (source.hasOwnProperty(key)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)